connectors/katalog/pkg/connector/connector.go

```diff
@@ -121,6 +130,7 @@ func (r *Handler) createAsset(c *gin.Context) {
 		assetPrefix = request.DestinationAssetID + "-"
```

Comment:
The validation should be done here (line 130) and not in the line 133

Reply:
Look at the following code:
	```
        assetPrefix := FybrikAssetPrefix
	if request.DestinationAssetID != "" {
		assetPrefix = request.DestinationAssetID + "-"
	}
	assetPrefix = r.conformeName(assetPrefix)
```
either request.DestinationAssetID exists or not, in both cases assetPrefix will end with "-" which fails the conform check. The check should be done on request.DestinationAssetID itself.

Reply:
Ok I see, "-" is not allowed in the end. I don't understand then whey we added this "-" to begin with, let's discuss tomorrow.